src/main.cpp

```diff
@@ -828,7 +831,9 @@ static bool run_dedicated_server(const GameParams &game_params, const Settings &
 
 	// Database migration
 	if (cmd_args.exists("migrate"))
-		return migrate_database(game_params, cmd_args);
+		return migrate_map_database(game_params, cmd_args);
+	else if (cmd_args.exists("migrate-players"))
+		return ServerEnvironment::migratePlayersDatabase(game_params, cmd_args);
```

Comment:
i moved this to ServerEnvironment as players are part of the environment and make access to database factory private. I can re-set it to main if needed, but i don't think in CPP a main should have many anonymous functions if not needed